src/Language/PureScript/CoreFn/FromJSON.hs

```diff
@@ -12,14 +12,14 @@ import           Data.Aeson
 import           Data.Aeson.Types (Parser, Value, listParser)
 import           Data.Text (Text)
 import qualified Data.Text as T
-import           Text.ParserCombinators.ReadP (readP_to_S)
 import qualified Data.Vector as V
-import           Data.Version (Version, parseVersion)
+import           Data.Version (Version)
 
 import           Language.PureScript.AST.SourcePos (SourceSpan(SourceSpan))
 import           Language.PureScript.AST.Literals
 import           Language.PureScript.CoreFn.Ann
 import           Language.PureScript.CoreFn
+import           Language.PureScript.Docs.Types (parseVersion')
```

Comment:
That dependency seems the wrong way around. Could we move that `parseVersion'` somewhere both the Docs and CoreFn can import from?

Reply:
No problem. Do you have any suggestions on where to put it?

Reply:
I've looked around a bit and I can't come up with anything better than just putting it into `CoreFn.FromJSON`. @hdgarrood any preferences?

Reply:
I feel like we should just dump this sort of stuff - stuff we kind of wished was provided by upstream libraries - into a Language.PureScript.Utils module. There are some modules in the compiler already for this sort of stuff which don’t use the Language.PureScript namespace but they really ought to, imo. But that’s probably separate... for now I don’t mind having one of these import the other. I don’t have a preference for which way around they are.

Reply:
Allright, in that case I'd prefer the function lived here. I see ide/docs/publish as downstream from the compilers PoV.